src/dmd/json.d

```diff
@@ -797,19 +797,33 @@ public:
         objectEnd();
     }
 
+    private void generateModules(Modules* modules)
```

Comment:
even private? 
from https://dlang.org/dstyle.html: 
`All public declarations will be documented in Ddoc format and should have at least Params and Returns sections.` => says public

Reply:
If they're non-trivial, yes. I don't know what `generateModules()` does. How can it generate a module when it is passed modules? Where do these generated modules go? I'm sure this is obvious to you since you wrote it, but I submit if you looked at it a year from now, you wouldn't know what it did. (This happens to me all the time.)

Reply:
I'm not questioning whether it should have a comment, I'm questioning whether it should have a comment formatted as Ddoc